### ISSUE TITLE
useLazyLoad: Undocumented behavior

### ISSUE THREAD

**Issue by weaverryan** [stimulus-use/stimulus-use#118]
Hi again!

I was going through the source tonight and noticed `useLazyLoad`. It's not documented or in the playground. Is that simply accidental? Is it functional and usable?

Thanks!

**Comment by adrienpoly** [comment]
Yes it is functional while very basic compared to alternative such as (https://github.com/verlok/vanilla-lazyload). It is a derivative of `useIntersection` using the `IntersectionObserver` API. 
I built it at the very start of this library mostly for testing the Observers. I never officially released it as I am not convinced this is still a relevant feature to have when most modern browser (except Safari) have the built-in feature `loading="lazy".

If you want to test it here is a basic html that should work

```html
<img data-controller="lazy-load" data-src="some-image-url">
```

**Comment by evenreven** [comment]
Since the Safari devs are being so slow with this, I'd argue that this is a relevant feature, and I'd love to use this as a composable behaviour. Also, Firefox dev seem to have no rush implementing loading=lazy for iframes, which is almost more important than for images if your site embeds YouTube videos. (I think it's borderline hilarious that Google (through Lighthouse) punishes devs for using Google products (i.e. YouTube, which is so full of bad scripts and placeholder images Lighthouse docks you several points if you don't lazyload the iframe).)

**Comment by marcoroth** [comment]
Resolved via https://github.com/stimulus-use/stimulus-use/commit/7f20a8abd2f795d88df4fcc16a97a20ea71f3224 and https://github.com/stimulus-use/stimulus-use/commit/480d0546837ad928f3cc05cf86c2dab4a1df9eaa
